doc/user-guide/github.md

```diff
@@ -125,3 +125,53 @@ If workflow needs an update a relevant message will be displayed:
 
 Please review the workflow file and update if needed.
 
+## Secrets
+
+To avoid uploading an `.env` file to Github, you may define them in your `ploomber-cloud.json` and Ploomber Cloud will read them from your environment variables. Here are the steps:
+
+1. Set secrets as environment variables using `export key=value`
+2. Define secret keys in `ploomber-cloud.json` under `secret-keys`
+3. Define the secrets in your Github actions config `.yaml` file
+4. Push your code to deploy
+
+For example, if I had two secrets, `key1` and `key2`, I would first set them as environment variables:
+
+```sh
+export key1=val1
+export key2=val2
+```
+
+Then I would edit my `ploomber-cloud.json` to look like this:
+
+```json
+"id": "project-id-1999",
```

Comment:
you're missing the outer `{}` characters, add them, otherwise people might copy-paste this and end with an invalid config file

Reply:
Fixed [here](https://github.com/ploomber/doc/pull/212/files#diff-bdba8f4b461f494edc3b95709bb0d5d869d0d6d1593c34c1ac790c9f24787e64R233-R239) and [here](https://github.com/ploomber/doc/pull/212/files#diff-f41679bbab33b86f374ce79abb5adcfbf4e6c05394c594cba25012bc1f48f43eR138-R144)